packages/@n8n_io/eslint-config/package.json

```diff
@@ -14,7 +14,8 @@
     "eslint-plugin-import": "~2.26",
     "eslint-plugin-n8n-local-rules": "~1.0",
     "eslint-plugin-prettier": "~4.2",
-    "eslint-plugin-vue": "~7.17"
+    "eslint-plugin-vue": "~7.17",
+    "tslint": "^6.1.3"
```

Comment:
No action to take in this PR, but I've wanted to bin TSLint for long but can never find the time - I was wondering if it would it be doable to just remove TSLint and later add back any rules from ESLint as we notice any to be missing over time? Most of the reasonable ones should already be covered by ESLint.

Reply:
I've also been trying to slowly get rid of tslint, but it'll be a while before we are done. until then I prefer to keep it around, as we have vast majority of eslint rules turned off in the UI code. So, until we enable those eslint rules, tslint is needed.